src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -133,7 +133,8 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
           break;
         }
         case Deletion: {
-          commitDeletion(effectfulFiber);
+          const safely = isHandlingError;
+          commitDeletion(effectfulFiber, safely);
```

Comment:
Just an observation that this is the only thing in the first pass that might throw.